soapui-testserver-api/src/test/resources/test-recipes/request-with-base64-attachment.json

```diff
@@ -0,0 +1,17 @@
+{
+  "testSteps": [
+    {
+      "type": "REST Request",
+      "method": "GET",
+      "URI": "http://google.com/",
+      "attachments" : [
+        {
+          "contentType" : "text/plain",
+          "name" : "A Name",
+          "contentId" : "An id",
+          "content": "Q29udGVudA=="
```

Comment:
Another thing that would be nice is to not have to specify content ID, and maybe autodetect the content type if it isn't specified. But let's do that later.